laravel-app/app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -70,7 +70,7 @@ public function register(CreateUserRequest $request): View
             'username' => $validatedData['username'],
         ]);
 
-        $user->user_information()->create([
+        $user->userInformation()->create([
```

Comment:
i dont like the naming of this. $user->userinformation because of the redundancy naming sense . anyway you already integrate it.

Reply:
Noted sir. Thank you ❤️

Reply:
Renamed from userInformation to information sir, thank you ❤️